webpages/login.php

```diff
@@ -0,0 +1,92 @@
+<?php
+session_start();
+if (!empty($_SESSION["uid"]))
+{
```

Comment:
use format for if as
```php
if (condition) {
  block;
}
```
if the block have a single statement do not use curly braces at all, it increases readability